ota/salus.js

```diff
@@ -0,0 +1,95 @@
+const axios = require('axios');
+const url = 'https://eu.salusconnect.io/demo/default/status/firmware?timestamp=0';
+const assert = require('assert');
+const common = require('./common');
+const tar = require('tar-stream');
+
+/**
+ * Helper functions
+ */
+
+async function getImageMeta(modelId) {
+    const images = (await axios.get(url)).data.versions;
+    const image = images.find((i) => i.model === modelId);
+    assert(image !== null, `No image available for modelId '${modelId}'`);
+    return {
+        fileVersion: parseInt(image.version, 16),
+        url: image.url.replace(/^http:\/\//, 'https://'),
+    };
+}
+
+async function untar(tarStream) {
```

Comment:
I wonder if we should move this to common, as I expect it might be useful elsewhere...